src/java/main/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -520,6 +520,12 @@ public synchronized void setCurrentVote(Vote v){
     protected boolean quorumListenOnAllIPs = false;
 
     /**
+     * Keeps time taken for leader election in milliseconds. Sets the value to
+     * this variable only after the completion of leader election.
+     */
+    private long electionTimeTaken = -1;
```

Comment:
Thanks @eribeiro ,

LeaderMXBean, FollowerMXBean will be available only after the quorum leader election and the value won't be changed until next LE. I think, adding 'volatile' doesn't make any difference, right?

code reference:

https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/quorum/Leader.java#L417
https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/quorum/Follower.java#L70

    Leader.java
        zk.registerJMX(new LeaderBean(this, zk), self.jmxLocalPeerBean);

    Follower.java
        fzk.registerJMX(new FollowerBean(this, zk), self.jmxLocalPeerBean);